src/CommitmentsV2/SFA.DAS.CommitmentsV2/Data/Configuration/ApprenticeshipUpdateConfiguration.cs

```diff
@@ -9,6 +9,7 @@ public class ApprenticeshipUpdateConfiguration : IEntityTypeConfiguration<Appren
         public void Configure(EntityTypeBuilder<ApprenticeshipUpdate> builder)
         {
             builder.ToTable("ApprenticeshipUpdate").HasKey(e => e.Id);
+            builder.Property(e => e.Id).ValueGeneratedNever();
```

Comment:
Is this correct to add this? Wouldn't this prevent an Id being created for the newly inserted records?

Reply:
Removed them